distributed/client.py

```diff
@@ -2188,7 +2188,7 @@ async def _gather(self, futures, errors="raise", direct=None, local_worker=None)
                 "Cannot gather Futures created by another client. "
                 f"These are the {len(mismatched_futures)} (out of {len(futures)}) "
                 f"mismatched Futures and their client IDs (this client is {self.id}): "
-                f"{ {f: f.client.id for f in mismatched_futures} }"
+                f"{{f: f.client.id for f in mismatched_futures}}"
```

Comment:
It looks like this update results in a genuine behavior change (TIL) 

```python
In [1]: f"{ {i:i for i in range(10)} }"
Out[1]: '{0: 0, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9}'

In [2]: f"{{i:i for i in range(10)}}"
Out[2]: '{i:i for i in range(10)}'
```

Can we add an ignore or something to this line?

Reply:
Good catch, I wasn't aware of that! Fixed